samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -79,10 +79,21 @@ void BLEConnectivityManager::ConnectionHandler(bt_conn *conn, uint8_t conn_err)
 		return;
 	}
 
-	LOG_INF("Connected: %s", str_addr);
-
 	/* TODO: Add security validation. */
 
+	if (conn_err) {
```

Comment:
I'm wondering if we should always start recover after connection error. If attempt was triggered by reconnect probably we should, but in case it was first connection attempt and it failed, maybe we should call `device->mConnectedCallback` to inform application layer that the device cannot be added.

Reply:
Ok, it makes sense.